### DISCUSSION TITLE
为什么报错呢？

### DISCUSSION THREAD

**Discussion by Dr395** [ramonvc/freegpt-webui#123] [Q&A] [answered]
![屏幕截图 2023-07-19 141520](https://github.com/ramonvc/freegpt-webui/assets/83225509/485b483b-3188-4f10-b925-8fb0fa97eb16)

**Comment by ramon-victor** [comment]
This is when the web access API is unstable. Disabling web access it will work

**Reply by Dr395** [reply]
Thank you very much for your answer, but it still doesn't work
![屏幕截图 2023-07-19 141520](https://github.com/ramonvc/freegpt-webui/assets/83225509/ae02c42d-c795-4e06-a9ef-e0fd11c51b3e)

**Reply by ramon-victor** [reply, marked as answer]
Your connection probably has several blocks that are affecting the APIS. You can try using Proxy or VPN.

Everything working correctly in replit
[freegpt-webui repl](https://freegpt-webui.ramonvc.repl.co/chat/)

**Reply by Dr395** [reply]
Thanks a lot
